src/debugging/DockerDebugConfigurationProvider.ts

```diff
@@ -24,7 +24,18 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
     ) { }
 
     public provideDebugConfigurations(folder: WorkspaceFolder | undefined, token?: CancellationToken): ProviderResult<DebugConfiguration[]> {
-        return undefined;
+        const add: MessageItem = { title: 'Add Docker Files' };
+
+        // Prompt them to add Docker files since they probably haven't
+        window.showErrorMessage(
+            'To debug in a Docker container on supported platforms, use the command \"Docker: Add Docker Files to Workspace\", or click the button below.',

```

Comment:
Nit: if I recall, it's preferable not to refer to the relative positions of things in the UX, as it's not always obvious what that means when using assistive technologies (e.g. screen readers).  It may be better to refer to the button by its name.